src/agent/src/main.rs

```diff
@@ -379,14 +379,9 @@ async fn start_sandbox(
     #[cfg(feature = "guest-pull")]
     image::set_proxy_env_vars().await;
 
-    // - When init_mode is true, enabling the localhost link during the
-    //   handle_localhost call above is required before starting OPA with the
-    //   initialize_policy call below.
-    // - When init_mode is false, the Policy could be initialized earlier,
-    //   because initialize_policy doesn't start OPA. OPA is started by
-    //   systemd after localhost has been enabled.
+    // TODO: initialize earlier.
```

Comment:
Is this initialise earlier comment for a future PR and related to the init data discussions?

Reply:
This was just a reminder for me that we *could* initialize the Policy module earlier, if we wanted. For example, we could use Policy data instead of some of the config.rs data.

But at this point I am not aware of anyone planning to use Policy data instead of config.rs data, so I will remove this comment/reminder.